build.gradle

```diff
@@ -290,6 +292,17 @@ subprojects { subproj ->
 		classpath = project.sourceSets.main.compileClasspath + configurations.shadowed
 	}
 
+	if (subproj.name in osgiProjects) {
+		apply plugin: 'osgi'
+
+		jar {
+			manifest {
```

Comment:
Does this get merged with all of the configuration we already have [here](https://github.com/junit-team/junit5/blob/1fc39a7456cd8da3f5f8fad996090157b4d9f6eb/build.gradle#L529)?

Reply:
Also, wouldn't this also need an `onlyIf` block like the existing one?

```groovy
		onlyIf {
			project.generateManifest
		}
```

Reply:
yes, both gets merged as the second `jar` closure enhances the settings as nothing is overridden by setting the same property, using the same method.